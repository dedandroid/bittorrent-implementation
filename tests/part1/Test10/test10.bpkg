ident:58824b29a52f5102d712113f3d5141c994cdfe794999a9b70d91ec01a62e7e7023d224b6d4f7daee78f5407e69cd84da7ac9f8fcce4a9280a76f7ddd6aaee79525f1dedad1464c6e3cdb9b8053e3a78cd7da6b43c8904ff8cc357b5ce4f9b858f335e78a01b41acd6f2eb8a9ca272801b379af3a1ef28bffb1d697f512d3ad550ceab15c04e8fd7af408c0ed2b1c86182f2d07a0b89b605556d16be86f4e556749540f5c7e7dec232048b302240897fd042037cb53830b72bbb6eb8109990870d9000db50480ff3abe0999a92332ca293293f5809008032b124bbf4d27fe117441736f3f047079b9bf47e84104f157698dded0dd44dcd85879f61471863dd975643340094d515aad3934da65093e2a38e7c27cbca0dfa7cd1f2e984a17831bbf271a4d6ed3d8ba5c97a3fed055360f5367da438166910fd968c6696cea2f982fffa322488ea8d723fe968026a453c72b6ce8f3171b0e22120b88baf5e48bca6275b68cd97d79f8b0338ed73ccb786dad253a1038ea2d3dd6165ed8a9411afc711ac3afb8ad6db33c98a7150562f5e67f032a2e3cb996cd265dd62db8faed05d08fbade7970f3d293f6914593f710ce04dbfb964063445d7440585fb56c53a578163ac7b3af70ce40ea84949f0e2b3a3406fdda0797755c636e7f20e3e1e2b16b7584f8b8f3e4f4852f45028e360e77aec22cbd4a02ff674968e6b65ec5d3c71
filename:test010.data
size:512
nhashes:3
hashes:
	1db14bf710c0cc9837ed06c48dc7e83655b03dd8742aa289e610ab198146b273
	bec074b76607e20cb1126a07091a8d7c39b81d7234c70bce09ab993069e3abfd
	cbf401e6739625f4ab0102338c275e08ac3ecfa6ae29a6a7039dfda5940cbf66
nchunks:4
chunks:
	3d64d616d414f98295c145d4fe7d72f97086e59a3b44161fbdfcb5719e69c720,0,128
	acf421466d2f7cb22c22778b3b9662b0ceba7a3f72bf18c9ef766848c3b1a48c,128,128
	606d945ddc3deac4cc0cd9dc74f9f81a1fe612b8d517c6f17458215c85b8edfd,256,128
	5fa99cc76b3013495fd979036e3db15aa7c8a48e22e1960487aa5f51b16bed4f,384,128
