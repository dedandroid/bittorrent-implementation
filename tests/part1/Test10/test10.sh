i='10'
../pkgchecker test$i.bpkg -hashes_of 1db14bf710c0cc9837ed06c48dc7e83655b03dd8742aa289e610ab198146b273 > test$i.out

#Surpress output for not
diff test$i.out test$i.expected > /dev/null

if [ $? -eq 0 ]; then
        echo "Output matches"
else
        echo "Outputs do NOT match!"
fi
