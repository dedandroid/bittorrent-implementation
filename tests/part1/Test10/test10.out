3d64d616d414f98295c145d4fe7d72f97086e59a3b44161fbdfcb5719e69c720
acf421466d2f7cb22c22778b3b9662b0ceba7a3f72bf18c9ef766848c3b1a48c
606d945ddc3deac4cc0cd9dc74f9f81a1fe612b8d517c6f17458215c85b8edfd
5fa99cc76b3013495fd979036e3db15aa7c8a48e22e1960487aa5f51b16bed4f
